sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -244,8 +245,12 @@ export class ServiceBusClient {
       | MSITokenCredentials,
     options?: ServiceBusClientOptions
   ): ServiceBusClient {
-    host = String(host);
-    const tokenProvider = new AadTokenProvider(credentials);
-    return ServiceBusClient.createFromTokenProvider(host, tokenProvider, options);
+    if (isNode) {
+      host = String(host);
+      const tokenProvider = new AadTokenProvider(credentials);
+      return ServiceBusClient.createFromTokenProvider(host, tokenProvider, options);
+    } else {
+      throw new Error("`createFromAadTokenCredentials` is not supported in browser.");
```

Comment:
The error message can be improved by alluding to the fact that AAD support is not yet present in browser mode and so `createFromAadTokenCredentials` method cannot be used to create ServiceBusClient